internal/clients/etherscan/getbalance.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Finobo
```

Comment:
`balance.go` or `get-balance.go` would follow the naming convention. i think `balance.go` is better primarily because its shorter and still relays the majority of the information.